docs/reference/query-languages/esql/kibana/definition/functions/st_simplify.json

```diff
@@ -0,0 +1,85 @@
+{
+  "comment" : "This is generated by ESQL's AbstractFunctionTestCase. Do not edit it. See ../README.md for how to regenerate it.",
+  "type" : "scalar",
+  "name" : "st_simplify",
```

Comment:
I think this function should have preview:true. Probably need that in the StSimplify function-info.

Reply:
` "preview" : true` is already there at the end of the json, hence why it's difficult to spot